library/src/scala/runtime/stdLibPatches/Predef.scala

```diff
@@ -62,4 +62,8 @@ object Predef:
      *  `eq` or `ne` methods, only `==` and `!=` inherited from `Any`. */
     inline def ne(inline y: AnyRef | Null): Boolean =
       !(x eq y)
+
+  extension (inline opt: Option.type)
+    @experimental
+    inline def fromNullable[T](t: T | Null): Option[T] = Option(t).asInstanceOf[Option[T]]
```

Comment:
Either make `opt` non-`inline`, or actually use it in the body. Otherwise if the receiver contains side effects, they will be dropped by the current implementation.

Reply:
I don't understand why it is a problem if the receiver `opt` is dropped, since it should always be the `Option` object, and it  does not affect the logic?

Reply:
It could be `{ println("foo"); Option }`. Or any `getOption` where `def getOption: Option.type = { sideEffects(); Option }`.

Reply:
I see. I changed `opt` to non-inlined now.